Samples/PhotoEditor/cpp-winui-packaged/PhotoEditor.sln

```diff
@@ -0,0 +1,43 @@
+﻿
```

Comment:
We should rename the folder `PhotoEditor/cpp-winui` - packaged is the default so the guidelines are to not include "packaged" in the folder name unless we need to distinguish the sample with an unpackaged version.

Reply:
Fixed